plugins/main/bridge/bridge.go

```diff
@@ -358,103 +360,109 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
-	// run the IPAM plugin and get back the config to apply
-	r, err := ipam.ExecAdd(n.IPAM.Type, args.StdinData)
-	if err != nil {
-		return err
-	}
+	// Assume l2 interface only
```

Comment:
Please use uppercase L2 for consistency and legibility.

Reply:
Done